hack/make/test-integration-shell

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+bundle .integration-daemon-start
+bundle .integration-daemon-setup
+
+if ! grep integration-test-helpers /etc/bash.bashrc; then
+    echo "source $PWD/hack/make/.integration-test-helpers" >> /etc/bash.bashrc
+fi
```

Comment:
This bit will need to move into the `Dockerfile` -- these scripts are technically runnable on the host, and we don't want to be modifying `/etc/bash.bashrc` on someone's host without at least asking first. :smile: